release/ray_release/tests/test_config.py

```diff
@@ -217,7 +217,7 @@ def test_compute_config_invalid_ebs():
 
 
 def test_load_and_validate_test_collection_file():
-    read_and_validate_release_test_collection(_TEST_COLLECTION_FILE)
+    read_and_validate_release_test_collection([_TEST_COLLECTION_FILE])
```

Comment:
1. why is this not checking the result?
2. could you expand this test to read more than 1 files?